docs/source/models/ssdlite.rst

```diff
@@ -15,9 +15,9 @@ Model builders
 
 The following model builders can be used to instantiate a SSD Lite model, with or
 without pre-trained weights. All the model builders internally rely on the
-``torchvision.models.detection.ssd.SSD`` base class. Please refer to the `source
+``torchvision.models.detection.ssdlite`` base class. Please refer to the `source
```

Comment:
The base class is still `ssd.SSD`, but OK to change the link to `ssdlite.py` below.

```suggestion
``torchvision.models.detection.ssd.SSD`` base class. Please refer to the `source
```